web/themes/custom/pineapple/components/_card.twig

```diff
@@ -15,7 +15,12 @@
   <div class="card-content u-o-2">
     {% if labels %}
       <div class="card-labels">
-        {{ labels }}
+        {% for label in labels %}
```

Comment:
Consider using the word "item" instead of "label", looks odd doing `label.label`

Reply:
done